scripts.py

```diff
@@ -417,23 +418,48 @@ def do_run(self):
         work_ids_to_delete = set()
         unset_work_id = dict(work_id=None)
 
-        if self.force:
-            self.clear_existing_works()                  
+        if self.identifiers:
```

Comment:
This is the core of my change. If you specify a set of identifiers on the command line, you're asking that the Works for the LicensePools corresponding to those identifiers be recalculated from scratch.